modules/database-mysql/src/main/java/eu/cloudnetservice/modules/mysql/MySQLDatabase.java

```diff
@@ -224,7 +224,7 @@ public void iterate(@NonNull BiConsumer<String, JsonDocument> consumer) {
   @Override
   public @Nullable Map<String, JsonDocument> readChunk(long beginIndex, int chunkSize) {
     return this.databaseProvider.executeQuery(
-      String.format("SELECT * FROM `%s` ORDER BY ? LIMIT ? OFFSET ?;", this.name),
```

Comment:
How did this ever work?

Reply:
Interestingly, MariaDB responds with information for the field, so this was not noticed in the (unit) tests either 🤷